README.md

```diff
@@ -7,3 +7,26 @@
 
   ⚠️ This repository is currently a WIP 🔧
 </div>
+
+# Requirements
+
+- Elixir 1.16.1
+- Mix 1.16.1
+- Docker(optional)
+
+# Development Guide
+
+A PostgreSQL database is needed, using Docker and using this setup for the developing stage:
+
+```sh
+docker run \
+        --name lambcast_db \ 
```

Comment:
```suggestion
        --name lambcast_db \
```

There's a trailing space on this line, which breaks the command in two. This cause an error when copy-pasting the command